app/src/main/java/org/p2p/wallet/newsend/ui/search/adapter/SearchViewHolder.kt

```diff
@@ -65,16 +66,22 @@ class SearchViewHolder(
     private fun renderAddressOnly(item: SearchResult.AddressFound) {
         with(binding) {
             val imageIconUrl = item.sourceToken?.iconUrl
+            val description: String?
             val imageObject: Any = if (imageIconUrl != null) {
                 frameWalletImageView.setPadding(0, 0, 0, 0)
-
+                description = getString(
+                    R.string.search_no_other_tokens_description,
+                    item.sourceToken.tokenSymbol
+                )
                 imageIconUrl
             } else {
                 frameWalletImageView.setPadding(iconPadding, iconPadding, iconPadding, iconPadding)
                 R.drawable.ic_search_wallet
+                description = null
```

Comment:
no, it's broken
R.drawable.ic_search_wallet should be last because it's the result for imageObject

Reply:
-